src/components/Timeline/Playbook/DisplayPlaybook.js

```diff
@@ -10,17 +12,59 @@ export const DisplayPlaybook = ({
     incidentId
 }) => {
     let localKey = 0
-    return actions.map(action => <div key={localKey++}>
-        <span>
-            {action.name}
-        </span>
-        <br/>
-        <Play
-            action={action}
-            eventTypeId={eventTypeId}
-            eventId={eventId}
-            incidentId={incidentId}
-            ticketId={ticketId}
-            engagementId={engagementId}
-        />
-    </div>)}
+    return <div>
+        {actions.map(action =>
+            <div key={localKey++}>
+                <span>
+                    {action.name}
+                </span>
+                <br/>
+                <Play
+                    action={action}
+                    eventTypeId={eventTypeId}
+                    eventId={eventId}
+                    incidentId={incidentId}
+                    ticketId={ticketId}
+                    engagementId={engagementId}
+                />
+            </div>
+        )}
+    </div>
+}
+
+
+export const mapStateToDisplayPlaybookProps = (state, ownProps) => {
+    const auth = state.auth
+
+    const eventType = state.eventTypes.records[ownProps.eventTypeId]
+    const event = Object.values(state.events.list)
+        .find(event => event.id == ownProps.eventId)
```

Comment:
Should we be using === instead of == to avoid unexpected stuff like '' == 0 being true?